Modules/Common/src/EverIncreasingGraph.cxx

```diff
@@ -0,0 +1,83 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   EverIncreasingGraph.cxx
+/// \author Barthelemy von Haller
+///
+
+#include "Common/EverIncreasingGraph.h"
+
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+
+// ROOT
+#include <TGraph.h>
+#include <TH1.h>
+#include <TList.h>
+#include <TPaveText.h>
+
+using namespace std;
+
+namespace o2::quality_control_modules::common
+{
+void EverIncreasingGraph::configure(std::string /*name*/) {}
+
+Quality EverIncreasingGraph::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
+{
+  auto mo = moMap->begin()->second;
+  Quality result = Quality::Good;
+  auto* g = dynamic_cast<TGraph*>(mo->getObject());
+
+  // simplistic and inefficient way to check that points are always increasing
+  int nbPoints = g->GetN();
+  double lastY = -DBL_MAX;
```

Comment:
Consider this, if you agree it is more clear and safer:
```
std::numeric_limits<double>::lowest
```
https://en.cppreference.com/w/cpp/types/numeric_limits/lowest says it actually returns `-DBL_MAX`

Reply:
thanks, this is nice.